tsl/src/compression/compression.c

```diff
@@ -488,6 +496,25 @@ compress_chunk_populate_sort_info_for_column(Oid table, const ColumnCompressionI
 	ReleaseSysCache(tp);
 }
 
+static void
```

Comment:
```suggestion
/* Run VACUUM ANALYZE on the given chunk. */
static void
```

Reply:
Maybe it should be called `run_vacuum_analyze` if that's what it does?

Reply:
Updated to `run_analyze_on_chunk`.